src/jsutils/instanceOf.ts

```diff
@@ -1,5 +1,14 @@
 import { inspect } from './inspect';
 
+/* c8 ignore next 4 */
+const isProduction =
+  // eslint-disable-next-line no-undef
+  typeof process !== 'undefined' &&
+  // eslint-disable-next-line no-undef
+  process.env &&
```

Comment:
We can remove this to expedite merging this, this issue should only occur when we have a DOM-node that has an `id` property with the value `process`

```suggestion
  // eslint-disable-next-line no-undef
  process.env &&
```